dvrk_psm:
 PSM1:
  outer_yaw_joint_controller:
    p: 20
    i: 0
    d: 10
  outer_pitch_joint_1_controller:
    p: 15
    i: 1
    d: 15
  outer_insertion_joint_controller:
    p: 100
    i: 0.1
    d: 15
  outer_roll_joint_controller:
    p: 40
    i: 0.1
    d: 15

 PSM2:
  outer_yaw_joint_controller:
    p: 20
    i: 0
    d: 10
  outer_pitch_joint_1_controller:
    p: 15
    i: 1
    d: 15
  outer_insertion_joint_controller:
    p: 100
    i: 0.1
    d: 15
  outer_roll_joint_controller:
    p: 40
    i: 0.1
    d: 15

 PSM3:
  outer_yaw_joint_controller:
    p: 20
    i: 0
    d: 10
  outer_pitch_joint_1_controller:
    p: 15
    i: 1
    d: 15
  outer_insertion_joint_controller:
    p: 100
    i: 0.1
    d: 15
  outer_roll_joint_controller:
    p: 40
    i: 0.1
    d: 15
